README.md

```diff
@@ -14,7 +14,21 @@ the [individual](https://github.com/mrc-ide/individual) package.
 
 Please note, malariasimulation is only compatible with R >= 4.0.0
 
-The package can be installed from github using the "remotes" library
+You can install the binary package on Windows and Mac using the following
+command:
+
+```R
+# Enable this universe
+options(repos = c(
```

Comment:
Giovanni, can you add a comment to explain why you need to enable the universe? Is this just an alternative option to using remotes or is it necessary?

Reply:
The line below explains that the remotes method requires RBuildTools. Is there something else I can add here?

Reply:
Oh - I installed it github using remotes, but I never ran the options lines 22-24 to enable the universe. Wondering if that is needed or not

Reply:
the instructions for the remotes are below. They don't require universe. Perhaps it's easier to see on the [rendered README](https://github.com/mrc-ide/malariasimulation/tree/doc/outputs#installation)?